device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsSessionDeviceOperation.java

```diff
@@ -29,14 +29,16 @@
 
     private List<UUID> cbsCorrelationIdList = Collections.synchronizedList(new ArrayList<UUID>());
 
+    private SubscriptionMessageRequestSentCallback subscriptionMessageRequestSentCallback;
+
     /**
      * Create logical device entity to handle all operation.
      *
      * @param deviceClientConfig the configuration of teh device.
      * @param amqpsDeviceAuthentication the authentication object associated with the device.
```

Comment:
Update constructor comments to reflect new parameter.

Reply:
Good catch